jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -294,12 +295,20 @@ const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = (
   if (!recording) {
     return <div>No recording</div>
   }
+  if (!sortingInfo) {
+    return <div>No sorting info</div>
+  }
+  if (!recordingInfo) {
+    return <div>No recording info</div>
+  }
   
   return (
     <div ref={divRef} className="PluginComponentWrapper" style={{position: 'absolute', left: 0, right: 0, top: 0, bottom: 0}}>
       <plugin.component
         sorting={sorting}
+        sortingInfo={sortingInfo}
```

Comment:
For convenience, sortingInfo and recordingInfo are passed in as props to SortingView plugins. As mentioned, these are no longer properties of Sorting and Recording.